src/api/data/repos.go

```diff
@@ -0,0 +1,18 @@
+package data
+
+import (
+	"github.com/kubernetes-helm/monocular/src/api/swagger/models"
+)
+
+// Repo is a Zoom Model for storing repositories
+type Repo models.Repo
+
+// ModelId returns the unique name of the Repo
+func (r *Repo) ModelId() string {
```

Comment:
Yeah, linter complained about it too, but unfortunately this is how they are named in Zoom https://github.com/albrow/zoom#what-is-a-model